db/migrate/20231013114720_add_item_type_to_meeting_agenda_item_journal.rb

```diff
@@ -0,0 +1,5 @@
+class AddItemTypeToMeetingAgendaItemJournal < ActiveRecord::Migration[7.0]
+  def change
+    add_column :meeting_agenda_item_journals, :item_type, :integer, limit: 1
```

Comment:
Same here I guess, even though it might not be as relevant for the existing journals

Reply:
Added.